.github/ISSUE_TEMPLATE/blocking_bug_report.yml

```diff
@@ -49,14 +49,10 @@ body:
           required: false
         - label: Windows amd64
           required: false
-        - label: Windows arm64
-          required: false
         - label: Linux amd64
           required: false
         - label: Linux arm64
           required: false
-        - label: MacOS amd64
-          required: false
```

Comment:
Good catch. If we have only arm64 Mac binaries, we can't support intel macs.

Reply:
Thanks goes to @MatthiasHertel80  :-)